source/bao_hyp/config.rst

```diff
@@ -529,6 +529,46 @@ However, this mechanism may not take effect if the physical mapping feature is e
 specific memory region. Cache coloring exclusively operates in virtual memory systems, i.e.,
 systems featuring Memory Management Units (MMUs) for address translation.
 
+Shared Memory Configuration
+---------------------------
+.. _Shared Memory Configuration:
+
+In Bao's configuration multiple (``shmem0``, ``shmem1``, ...) or no shared memory can be set. If
+at least one shared memory is defined, an  ID is assigned to which the shared memory is
+associated. Later, this ID can be associated with an IPC in the multi-guest configuration.
+
+.. figure:: img/shmem.svg
+    :align: center
+    :name: shmem-fig
+
+|
+
+The configuration of shared memories is done by populating a struct called
+``shmem``, as follows:
```

Comment:
```suggestion
The configuration of shared memoy is done by populating a struct called
``shmem``. The struct definition is presented bellow:
```

Reply:
Thanks! Fixed in commit https://github.com/bao-project/bao-docs/pull/78/commits/b4d14d8ed94fcbddc9aa54c21ecef8d0eb53111b.